sql/updates/world/2018_04_18_01_world.sql

```diff
@@ -0,0 +1,2 @@
+-- Quest 28000 - Do the Imp-possible
```

Comment:
impsy is npc 47339.. this npc has to receive the script.. not the credit.. 47339 send on spell hit the credit spell..

Reply:
I was able to attack the imp with any spell, he didn't die, but just gave me a "Right in the face! Ow ow ow..." line, then I bagged him up.

Reply:
His health won't drop below 1 or 0.1% (regardless of what you do while on this quest),